DiscordChatExporter.Cli/Commands/Base/ExportCommandBase.cs

```diff
@@ -112,6 +126,14 @@ protected async ValueTask ExecuteAsync(IConsole console, IReadOnlyList<Channel>
             );
         }
 
+        // Assets directory should only be specified when the download assets option is set.
```

Comment:
No period at the end of single-line comments (consistency)

```suggestion
        // Assets directory should only be specified when the download assets option is set
```